helm-charts/dnsbl-exporter/values.yaml

```diff
@@ -0,0 +1,339 @@
+## @param replicaCount Number of Aplication replicas to deploy
+## NOTE: ReadWriteMany PVC(s) are required if replicaCount > 1
+##
+replicaCount: 1
+## @param updateStrategy.type Application deployment strategy type
+## ref: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy
+## NOTE: Set it to `Recreate` if you use a PV that cannot be mounted on multiple pods
+## e.g:
+## updateStrategy:
+##  type: RollingUpdate
+##  rollingUpdate:
+##    maxSurge: 25%
+##    maxUnavailable: 25%
+##
+updateStrategy:
+  type: RollingUpdate
+## @param priorityClassName powerdnsAdmin pod priority class name
+##
+priorityClassName: ""
+## @param schedulerName Name of the k8s scheduler (other than default)
+## ref: https://kubernetes.io/docs/tasks/administer-cluster/configure-multiple-schedulers/
+##
+schedulerName: ""
+## @param topologySpreadConstraints Topology Spread Constraints for pod assignment
+## https://kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/
+## The value is evaluated as a template
+##
+topologySpreadConstraints: []
+## @param hostAliases Application pod host aliases
+## https://kubernetes.io/docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases/
+##
+hostAliases: []
+## @param extraVolumes Optionally specify extra list of additional volumes for powerdnsAdmin pods
+##
+extraVolumes: []
```

Comment:
Part of me thinks that `extraVolumes` and `extraVolumeMounts` could also be removed. Along with the logic in the deployment. But let me know what your thoughts are. Unless this is just standard helm boilerplate.

Reply:
By default they are not being used, but may be very helpful for debugging. As I said earlier, the helm chart creates `targets.ini` and `rbls.ini` as a config map. May be someone has other own application and would have these configs on a separate volume/etc - who knows.

Why not to be flexible?

Reply:
Yes, you are correct.